app/src/main/java/org/schabi/newpipe/MainActivity.java

```diff
@@ -92,11 +92,13 @@
     private boolean servicesShown = false;
     private ImageView serviceArrow;
 
-    private static final int ITEM_ID_SUBSCRIPTIONS = - 1;
-    private static final int ITEM_ID_FEED = - 2;
-    private static final int ITEM_ID_BOOKMARKS = - 3;
-    private static final int ITEM_ID_DOWNLOADS = - 4;
-    private static final int ITEM_ID_HISTORY = - 5;
+
+    private static final int ITEM_ID_HOME = 2;
+    private static final int ITEM_ID_SUBSCRIPTIONS = 3;
+    private static final int ITEM_ID_FEED = 4;
+    private static final int ITEM_ID_BOOKMARKS = 5;
+    private static final int ITEM_ID_DOWNLOADS = 6;
+    private static final int ITEM_ID_HISTORY = 7;
     private static final int ITEM_ID_SETTINGS = 0;
     private static final int ITEM_ID_ABOUT = 1;
```

Comment:
While you are at it, let's also reorder things a little bit ;-)
```suggestion
    private static final int ITEM_ID_HOME = 0;
    private static final int ITEM_ID_SUBSCRIPTIONS = 1;
    private static final int ITEM_ID_FEED = 2;
    private static final int ITEM_ID_BOOKMARKS = 3;
    private static final int ITEM_ID_DOWNLOADS = 4;
    private static final int ITEM_ID_HISTORY = 5;
    private static final int ITEM_ID_SETTINGS = 6;
    private static final int ITEM_ID_ABOUT = 7;
```

Reply:
Done!